source/stylesheets/_refills.scss

```diff
@@ -0,0 +1,16 @@
+.refill {
+  border-top: 2px solid $base-border-color;
+  margin-bottom: $base-spacing * 6;
+  padding-top: $base-spacing;
+}
+
+.refill-wrapper {
+  align-items: center;
+  background-color: tint($base-border-color, 90%);
+  border: $base-border;
+  display: flex;
+  justify-content: space-around;
+  margin-top: $base-spacing;
+  min-height: $base-spacing * 12;
```

Comment:
I tend to agree, but was trying to be consistent per one of @tysongach's comments on a previous PR:

> Why 6em? Can we make a more robust spacing system based on `$base-spacing`?

I don't want to get into setting a bunch of spacing variables, I'd rather keep everything with stock Bitters.